src/server/api/routers/voters.ts

```diff
@@ -47,6 +47,7 @@ export const votersRouter = createTRPCRouter({
             .findMany({
               where: {
                 roundId: ctx.round.id,
+                ballot: { publishedAt: { not: null } },
```

Comment:
Will this affect showing the voters list in the admin ui?

Reply:
It won't ! Only affects the check-mark icon

Reply:
@carlbarrdahl Here is an example
![image](https://github.com/user-attachments/assets/6273672b-c909-4a51-8218-faafd9f53577)

Reply:
Amazing, great work!